manifests/params.pp

```diff
@@ -2,59 +2,64 @@
 class foreman_proxy_content::params {
 
   # when not specified, we expect all in one installation
-  $parent_fqdn                  = $facts['fqdn']
-
-  $reverse_proxy                = false
-  $reverse_proxy_port           = 8443
-
-  $ssl_protocol                 = undef
-
-  $certs_tar                    = undef
-  $rhsm_hostname                = undef
-  $rhsm_url                     = '/rhsm'
-
-  $puppet                       = true
-
-  $pulp_admin_password          = extlib::cache_data('foreman_cache_data', 'pulp_node_admin_password', extlib::random_password(32))
-  $pulp_max_speed               = undef
-  $pulp_num_workers             = undef
-  $pulp_proxy_password          = undef
-  $pulp_proxy_port              = undef
-  $pulp_proxy_url               = undef
-  $pulp_proxy_username          = undef
-  $pulp_puppet_wsgi_processes   = 1
-  $pulp_ca_cert                 = undef
-  $pulp_worker_timeout          = 60
-
-  $qpid_router                  = true
-  $qpid_router_agent_addr       = undef
-  $qpid_router_agent_port       = 5647
-  $qpid_router_broker_addr      = 'localhost'
-  $qpid_router_broker_port      = 5671
-  $qpid_router_hub_addr         = undef
-  $qpid_router_hub_port         = 5646
-  $qpid_router_logging_level    = 'info+'
-  $qpid_router_logging          = 'syslog'
-  $qpid_router_logging_path     = '/var/log/qdrouterd'
-  $qpid_router_ssl_ciphers      = undef
-  $qpid_router_ssl_protocols    = undef
-  $qpid_router_sasl_mech        = 'PLAIN'
-  $qpid_router_sasl_username    = 'katello_agent'
-  $qpid_router_sasl_password    = extlib::cache_data('foreman_cache_data', 'qpid_router_sasl_password', extlib::random_password(16))
-
-  $enable_ostree                = false
-  $enable_yum                   = true
-  $enable_file                  = true
-  $proxy_pulp_isos_to_pulpcore  = true
-  $enable_puppet                = true
-  $enable_docker                = true
-  $enable_deb                   = true
-
-  $manage_broker                = true
-
-  $pulpcore_manage_postgresql   = true
-  $pulpcore_postgresql_host     = 'localhost'
-  $pulpcore_postgresql_port     = 5432
-  $pulpcore_postgresql_user     = 'pulp'
-  $pulpcore_postgresql_password = extlib::cache_data('pulpcore_cache_data', 'db_password', extlib::random_password(32))
+  $parent_fqdn                     = $facts['fqdn']
```

Comment:
I've started to stop aligning all parameters because it creates a lot of noise in git logs. Feel free to do the same.

Reply:
@ekohl I do find it's very nice to have them aligned for readability. Is it a better approach to do these alignment fixups in a separate commit in the future?

Reply:
Another thought, is it better still to keep the consistent alignment only within each block, when the number of parameters is large?

Reply:
yes, I think per block alignment makes much more sense and is a decent compromise between the two